src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -161,6 +161,43 @@
  * }
  * </pre>
  * <p>
+ * To configure the check with policy {@code alone} for
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_SWITCH">
+ * LITERAL_SWITCH</a> tokens:
```

Comment:
Change switch tokens (plus link) to switch statements. We don't want users to think this will handle switch expressions. Do the same for the others.

Reply:
Please check link once
Done